content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+relatedImages:
+- image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  name: ""
+- image: docker.io/example/example-operator-bundle:0.2.0
+  name: ""
+- image: docker.io/example/example-operator:0.2.0
+  name: ""
+schema: olm.bundle
+
+```
+</details>
+
+## Semver Veneer
+
+Since a `veneer` is identified as an input schema which is processed to generate a valid FBC, we can define a `semver veneer` as a schema which uses channel conventions to facilitate the auto-generation of channels adherent to [Semantic Versioning](https://semver.org/) (semver) guidelines.  
+
+[**DISCLAIMER:** since version build metadata [MUST be ignored when determining version precedence](https://semver.org/#spec-item-10) when using semver, it cannot be used in any bundle included in the `semver veneer` and will result in a fatal error.]
+
+### Schema Goals
+The `semver veneer` must have:
+- terse grammar to minimize creation/maintenance effort
+- deterministic output
+- simple channel promotion for maturing bundles
+- demonstration of a common type of channel maturity model
+- minor-version (Y-stream), major-version (X-stream) versioning optionality
+
+The resulting FBC must clearly indicate how generated channels relate to veneer entities.
+
+### Schema Anatomy
```

Comment:
WDYT about the titles be more friendly for the users like: When should I use it? How does it work? Examples and etc ..